metron-stellar/stellar-common/src/main/java/org/apache/metron/stellar/dsl/functions/StringFunctions.java

```diff
@@ -133,15 +131,15 @@ public Object apply(List<Object> strings) {
   }
 
   @Stellar( name="JOIN"
-          , description="Joins the components in the list of strings with the specified delimiter."
-          , params = { "list - List of strings", "delim - String delimiter"}
+          , description="Joins the non-null items in the iterable as strings with the specified delimiter. Null items are dropped."
+          , params = { "iterable - iterable (e.g. List, LinkedHashSet, etc.) of items treated as strings", "delim - String delimiter"}
           , returns = "String"
           )
```

Comment:
do we need to SurpressWarnings("unchecked")?

Reply:
> do we need to SurpressWarnings("unchecked")?

That's always been there - it's calling out the unchecked type cast `(Iterable<Object>) args.get(0);` (previously `(List<Object>) args.get(0);` Alternatively, we can add extra instanceof checks, but it's really 6 of one, half a dozen of the other. You get a classcastexception.

Reply:
Any thoughts on this @ottobackwards?

Reply:
I'm not saying it is new, but we can add the surpress while we are here.

Reply:
I'm sorry, I don't think I follow - line 140 just above the `apply(...)` method already has a "@SuppressWarnings" annotation. Are you asking whether we need to remove it, or were you suggesting adding another one somewhere?

Reply:
please, ignore this, sorry.